SPECS/kernel/config

```diff
@@ -1899,8 +1899,6 @@ CONFIG_ETHTOOL_NETLINK=y
 #
 # Device Drivers
 #
-CONFIG_HAVE_EISA=y
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/7c376f91f2e1686f3d0580fff0d3a49390c8e19a